Cabal/Distribution/PackageDescription/FieldGrammar.hs

```diff
@@ -542,3 +552,28 @@ setupBInfoFieldGrammar def = flip SetupBuildInfo def
     <$> monoidalFieldAla "setup-depends" (alaList CommaVCat) L.setupDepends
 {-# SPECIALIZE setupBInfoFieldGrammar :: Bool -> ParsecFieldGrammar' SetupBuildInfo #-}
 {-# SPECIALIZE setupBInfoFieldGrammar :: Bool ->PrettyFieldGrammar' SetupBuildInfo #-}
+
+-------------------------------------------------------------------------------
+-- Field formatters - Define how field values should be formatted for 'pretty'.
```

Comment:
This comment is not right. These functions also tell how fields are parsed as well. There is `unpack` for them as well.

Reply:
Any suggestions for a better name which encompasses both "format" and "unpack"? Or maybe just change the comment to:

"Define how field values should be formatted and parsed."

And rename the functions to something like `annotateHsSourceDirs` (name tbd)?

Reply:
I think `format` is ok name. The `hsSourceDirsGrammar` is also an option. I don't feel strongly about these (I'm thinking of a way for not needing them).

Reply:
Updated comment, going to leave names as is.